### DISCUSSION TITLE
[HELP] MariaDB commands for Postal to tune the sending speed via SSH

### DISCUSSION THREAD

**Discussion by bukunmilab** [postalserver/postal#3168] [Help with using Postal]
I accessed MariDB using this SSH command:

```
docker exec -it postal-mariadb sh
mariadb --user=user_name --password=your_password db_name
```

Note: I already use the scale worker command, so that is not what I am asking for.

Since I am running Postal in a docker with the default installation, can anyone share the MariaDB commands for Postal to tune the sending speed via SSH? command(s) that can write into the MariaDB configuration file or something like that.

**Comment by willpower232** [comment]
You can see the original recommendations here https://github.com/postalserver/postal/issues/324#issuecomment-323538575

As you're apparently using docker, you could put your changes in a file and use a docker volume to add that file to ` /etc/mysql/conf.d` in the container.
